automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestFunctions.scala

```diff
@@ -4,6 +4,7 @@ import org.clulab.aske.automates.TestUtils._
 
 class TestFunctions extends ExtractionTest {
 
+  // todo: how to test fragment (mentions with only input/output) extraction??
```

Comment:
You should be able to use the testTextBoundMention method?

Reply:
Oh, that was already taken care of by testUnaryEvent (in testFunctionFragments). I should delete that todo note. Thanks!